src/containers/pages/FocusInvestigation/active/index.tsx

```diff
@@ -453,6 +494,7 @@ const mapStateToProps = (state: Partial<Store>, ownProps: any): DispatchedStateP
   );
   return {
     caseTriggeredPlans,
+    jurisdictionParentId,
```

Comment:
What caused the need for this?

Reply:
> What caused the need for this?

I need to use it in the component when making the call to makePlansArraySelector()